src/install.ts

```diff
@@ -0,0 +1,35 @@
+import { App, reactive } from 'vue'
+import {
+  DriftWidget,
+  defaultConfig,
+  VueDriftWidgetConfig,
+  OptionalVueDriftWidgetConfig,
+} from './types'
+import { loadScript } from './utils'
+
+const VueDriftWidget: DriftWidget = {
+  install: (
+    app: App,
+    widgetId: string,
+    config: OptionalVueDriftWidgetConfig = defaultConfig,
```

Comment:
feels like those should be merged as setting an ID feels like part of the widget configuration..

Reply:
would it be bad practice to let this default to `process.env.VUE_APP_DRIFT_WIDGET_ID` if not set ?

Reply:
merging seems not be needed (i checked implementation) but docs state it's a function with two arguments...

still I think widgetId is part of widget config?

Reply:
```
process.env.VUE_APP_DRIFT_WIDGET_ID
```
is a webpack feature and will fail in vite or when using vue without vue-service-cli. Will not implement

Reply:
but still widget id should be part of the config?

Reply:
I like the id to be separated since it doesn't require you to construct an object in a first place.. the config is imho separated to id. However, to not sink into further discussions here, I changed it the way you'd like it to be.